CHANGELOG.rst

```diff
@@ -2,6 +2,7 @@
 CHANGELOG
 =========
 
+=======
```

Comment:
What was the reason for adding this? I'm not sure if it's going to affect the automated release notes.

Reply:
I didn't add that, to my knowledge. I'm gonna probably wind up redoing this entire PR from scratch... I have no idea how this all went so sideways.. So you can probably just stop looking at it. :)